### DISCUSSION TITLE
Angular Ivy | Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#23] [Announcements]
# Angular Ivy | Angular с примерами кода

Angular Ivy представляет собой абсолютно новый компилятор, который пришел на смену View Engine

https://angdev.ru/doc/angular-ivy/

**Comment by MaximZoz** [comment]
ɵrenderComponent и ɵmarkDirty нет в @angular/core v15, как можно пофиксить эту ошибку?
